warehouse/grpc_test.go

```diff
@@ -989,7 +989,45 @@ func TestGRPC(t *testing.T) {
 					})
 					require.NoError(t, err)
 					require.NotEmpty(t, res)
-					require.Equal(t, "Invalid destination creds, failed for operation: upload with err: \nEndpoint:  does not follow ip address or domain name standards.", res.GetError())
+					require.Equal(t, "Invalid destination creds, failed for operation: list with err: \nEndpoint:  does not follow ip address or domain name standards.", res.GetError())
+					require.False(t, res.GetIsValid())
+				})
```

Comment:
We can probably remove this test since we are already verifying it in the new one.